static/docs/commands-reference/gc.md

```diff
@@ -91,3 +99,5 @@ Let's check the size now:
 $ du -sh .dvc/cache/
 3.1G    .dvc/cache/
 ```
+
+<!-- TODO: Add `dvc gc -aT` example. -->
```

Comment:
let's not commit TODOs please :) create a ticket/checkbox for that

Reply:
Accidental commit. Removing...